### ISSUE TITLE
Deployment: Failing

### ISSUE THREAD

**Issue by jackdishman** [bostondao/nouns-monorepo-fork#28]
yarn run build in packages/nouns-webapp works fine and I can serve the build files locally using serve -s build. It's on Vercel that yarn run build is crashing:

```
[08:03:23.979] gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
[08:03:23.979] ../hidapi/linux/hid.c:44:10: fatal error: libudev.h: No such file or directory
[08:03:23.979]  #include <libudev.h>
[08:03:23.980]           ^~~~~~~~~~~
[08:03:23.980] compilation terminated.
[08:03:23.980] make: *** [Release/obj.target/hidapi-linux-hidraw/hidapi/linux/hid.o] Error 1
[08:03:23.981] gyp ERR! build error 
[08:03:23.981] gyp ERR! stack Error: `make` failed with exit code: 2
[08:03:23.981] gyp ERR! stack     at ChildProcess.onExit (/vercel/path0/node_modules/node-gyp/lib/build.js:194:23)
```

Goal here is to deploy the webapp on _any_ hosting service!

**Comment by jackdishman** [comment]
dotenv: 
```
# Active Chain ID
REACT_APP_CHAIN_ID=5

# If you have an Infura Project ID
REACT_APP_INFURA_PROJECT_ID=f11ef31e33694bf6a80958a8adfb1f77

```

**Comment by ghost** [comment]
👋 @jackdishman 

Wanted to ping you here as well in-case the discussions thread didn't notify you (ref: https://github.com/nounsDAO/nouns-monorepo/discussions/685#discussioncomment-6296888). 

**tl;dr:** I'm fairly sure the issue is system libraries being unavailable in the Vercel build environment, the comment I posted in the discussion should have pointers to get you going though. I hope it's helpful!
